hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/AbstractHoodieWriteClient.java

```diff
@@ -194,11 +196,14 @@ public boolean commitStats(String instantTime, List<HoodieWriteStat> stats, Opti
     }
 
     // callback if needed.
+    List<String> partitionPath = new ArrayList<>();
+    stats.forEach(stat -> partitionPath.add(stat.getPartitionPath()));
+
     if (config.writeCommitCallbackOn()) {
       if (null == commitCallback) {
         commitCallback = HoodieCommitCallbackFactory.create(config);
       }
-      commitCallback.call(new HoodieWriteCommitCallbackMessage(instantTime, config.getTableName(), config.getBasePath()));
+      commitCallback.call(new HoodieWriteCommitCallbackMessage(instantTime, config.getTableName(), config.getBasePath(), partitionPath.toString()));
```

Comment:
Is there a way to make the logic here extendable, so when the user want more information ,we don't need to make any change here.
such as  `new HoodieWriteCommitCallbackMessage(instantTime, config, ...)`

Reply:
good, let me think about it